lib/content-services/src/lib/search/services/search-header-query-builder.service.spec.ts

```diff
@@ -115,9 +141,12 @@ describe('SearchHeaderQueryBuilderService', () => {
             const expectedResult = [{ query: 'PARENT:"workspace://SpacesStore/fake-node-id"' }];
 
             const config: SearchConfiguration = {
-                categories: [{ id: 'cat1', enabled: true } as any, { id: 'cat2', enabled: true } as any],
+                categories: [
+                    { id: 'cat1', enabled: true },
+                    { id: 'cat2', enabled: true }
+                ],
                 filterQueries: expectedResult
-            };
+            } as SearchConfiguration;
```

Comment:
when you added as SearchConfiguration you don't need anymore type at left side - type of config const will be detected automatically, please remove redundant type there